### ISSUE TITLE
1.2.3 / 1.2.5 Audio description - right language required?

### ISSUE THREAD

**Issue by detlevhfischer** [w3c/wcag#3873]
As far as I can see, neither the normative text of 1.2.3 and 1.2.5 nor the definition of "audio description" has anything on the 'right' language for such description (maybe due to an english-centric bias where AD in different languages rarely surfaces). In a current audit of a German site, there are videos with a voice-over comment in English describing what happens.

Looking strictly at the normative text we'd have to pass this provided the AD covers the visuals sufficiently - even if the AD language were Chinese (and we'd be able to tell whether it covers the visuals, which we aren't) - right? 

Since there is no markup for spoken language, 3.1.2 Language of parts would not really help here.

Or am I missing something?

**Comment by mbgower** [comment]
**Proposed draft response**

You are correct. There is no wording in any of the 1.2 Time-based Media which makes an explicit statement that the alternative provided is in the same language as the original (with the possible exception of the language in 1.2.1 which uses the phrase "presents equivalent information").

This shortcoming is most apparent in situations where translation of time-based media content exists, in the form of subtitles or dubbing.
If a video is originally in English and has Spanish subtitles, do those qualify as captions, whic are defined as:
> synchronized visual and/or [text alternative](https://www.w3.org/WAI/WCAG21/Understanding/captions-prerecorded#dfn-text-alternative) for both speech and non-speech audio information needed to understand the media content

Anyone who speaks English but not Spanish, who is in need of captions, will not consider Spanish 'captions' to meet their need, in the above situation.

Given this problem has existed for 15 years, it may be unnecessary to make a normative change to the 2.x wording; however, the problem has been added to the [WCAG 3 parking lot ](https://github.com/w3c/wcag/wiki/WCAG-3-parking-lot-considerations-from-2.x#assumption-of-matching-languagedialect-is-not-explicit)

**Comment by detlevhfischer** [comment]
Thanks, Mike. I think the gap is clear and the need for right language of AD evident. 
I was just trying to frame it in terms of the practical problem of how to deal with it in a PASS/FAIL audit.

I was saying "we'd have to pass this provided the AD covers the visuals sufficiently".

When you replied, "You are correct", do you agree that we must pass this since there is no normative language to FAIL it?
Or do you say, "everyone knows that right language is implied, so we should FAIL this even in the absence of normative text supporting that?" Which of the two is it?

Whichever way this goes, I'd really like to get a WG consensus on this since this situation is not infrequent.
For cases of alt in the wrong language, we can at least resort to 2.4.4 (links) or 2.4.6 (button labels) and say "this is not decriptive" (for the intended audience). But bringing AD under 2.4.6 would surely be a stretch.

**Comment by patrickhlauke** [comment]
Agree that there is an unspoken assumption in the SCs (and even in 1.1.1 when it comes to text alternatives) that of course nobody would be so silly as to provide the alternatives/descriptions/etc in a completely different language from the surrounding content (not necessarily the page as a whole, but at least its immediate vicinity/context). While adding a requirement along those lines can't be done in non-normative documents (otherwise it would, quite rightly, be called out as adding new normative requirements by the backdoor), it would be good to at least strongly suggest that this should be done, but that it *technically* passes the tightly defined wording of the SC

**Comment by melaniephilipp** [comment]
Does the requirement that a conforming alternate version "provides all of the same information and [functionality](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#functiondef) in the same [human language](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#human-langdef)..." give you the normative cover you are looking for?

**Comment by dabrams888** [comment]
> of course nobody would be so silly as to provide the alternatives/descriptions/etc in a completely different language
I ran into this silly scenario the other day, though it wasn't as straight-forward as you may think. In testing a video that had both English and Spanish dialog in the audio track, I wasn't sure what the best recommendation would be for the language of captions and transcripts.

### Option 1
The captions/transcripts keep a consistent language throughout and provide a separate version for each language (including indications when the spoken language changes).

This seems more like a "subtitle" than a "caption".

### Option 2
The captions/transcripts match the exact text that was spoken regardless of language since that's more "equivalent" to the audio. 

Practically speaking, option 1 sounds better. But option 2 seems like the more "conformant" option since it's an exact alternative to the spoken audio.

**Comment by mbgower** [comment]
I think @melaniephilipp 's point about [conforming alternate version](https://www.w3.org/TR/WCAG22/#dfn-conforming-alternate-versions) is worth exploring, particularly note 3:
> If multiple language versions are available, then conforming alternate versions are required for each language offered.

That seems to potentially give us some scope to address the existing normative language of all the different use cases cited on this page.

**Comment by detlevhfischer** [comment]
As it stands, the proposed WG answer does not clarify my question, which was focused on whether the use of the wrong language in AD is a normative fail. So I think either the WG answer should clarify that technically speaking, it is not (as @patrickhlauke seems to suggest above) or this needs more time to develop an argument that finds normative backing for a FAIL elsewhere. I am fine with it either way, but what to get out of the grey zone.

**Comment by guyhickling** [comment]
Language mismatches don’t just happen in video captions and (as Patrick points out above) SC1.1.1 alternative texts). It also occurs in a variety of other kinds of content mark-up where two or more languages are involved, on bi-lingual websites. 

The problem concerns everything in these foreign language translations, including aria-label attributes, alt texts, texts hidden by CSS for screen readers, aria-roledescription attributes, the <title> element in the <head> section, and a few other things. These are far more common than captions in the wrong language (in fact I’ve never seen that in captions myself, though I can imagine someone would do it and then claim compliance), and I’ve been thinking about them for a while. For instance, I frequently have to audit Welsh page translations created for public sector organisations and agencies in the UK, and I almost never see these screen reader items translated. Which is not good for users who are not fluent in the original language.

So I have raised a separate GitHub issue for that, as I think those screen-reader things would need one SC, and this matter of captions and audio descriptions (and transcriptions as well) should have a separate SC. I do advocate for having a new SC for this, and I don’t think this captions/audio descriptions problem should have to wait for WCAG3 either.

**Comment by gundulaniemann** [comment]
I fully agree with the last sentences. Do we already have github issues for that, or some other measure this point does not get lost?

**Comment by alastc** [comment]
Worth getting agreement on this approach before starting that work, we can hang them off this issue.

**Comment by mbgower** [comment]
@gundulaniemann 

> Do we already have github issues for that, or some other measure this point does not get lost?

This issue will be discussed on tomorow's AGWG call. If we get support coalescing around one of the draft responses, we can then proceed to create PRs to resolve this issue (it would not be a draft response only, if the second draft got support).

**Comment by detlevhfischer** [comment]
I read yesterday's call minutes and the resoution on this issue.  
To be honest, I would not have raised it had I known that it (and the changes that will consequently be made to understanding texts) would take up so much time.

One thing would be important to clarify in those additions regarding the use of the correct language.

There are cases where a page in one language (say, German) contains a self-contained element in another language (say, English), for example, a video showing some activity with a running English commentary that may not cover some of the visual information, thus requiring an additional audio description. This AD could be in English (matching the language of the self-contained content element) or it could be in German (matching the language of the page). I think the addition should be clear about this situation - in the case sketched, personally I would find both approaches defensible and would not call either of them a FAIL of 1.2.3/1.2.5. 

The situation is clearly different for alt text, (hidden) element labels, error message language, etc., where the agreed addition to understanding texts would clarify that the language of those needs to match that of the page content (and failing to do so would likely cause a FAIL in 2.4.6 or 2.4.4).

**Comment by mbgower** [comment]
Here is my suggested boiler plate to add to the listed SCs (with potentially some minor additions/alterations based on context).

> ### Text alternatives and equivalents match the language of the content
>[5.2.4 Conformance](https://www.w3.org/TR/WCAG22/#cc4) language requires that success criteria be met through accessibility-supported ways, where  the technology is used “in the human language of the content.” In addition, where an alternative version is used, it is defined as something that “provides all of the same information and functionality in the same human language.”  Text alternatives and equivalents should match the human language of the original content (normally the default human language of the page).

**Comment by bruce-usab** [comment]
Discussed on Backlog call 3/7.  Concurrence that "in the same human language" is part of the requirement for Audio Description.  Removing issue from Project Board as we expect to close with PR # 4262.
